src/search/ui/place.directive.html

```diff
@@ -22,7 +22,7 @@
   </div>
 
   <div class="col-sm-2">
-    <span>permanent</span>
+    <span translate-once="search.periodicOpen"></span>
```

Comment:
A place can be either `permanent` or `periodic`. So we'll need to do a check here and then print either one.

Reply:
So this code never worked the way it was supposed to?

Reply:
Yes, that was the point of the ticket I believe. Someone that wrote this a long time ago probably assumed that places are just always "permanent". But you can double-check with @corneelwille if you want to be 100% sure.

Reply:
Yes, exactly. For places with calendarType `periodic`, the text should be `Periodiek open`, for those with calendarType `permanent` the text can remain `Permanent`

Reply:
After discussing with @corneelwille I visualize `periodic` dates in the same way like events